go/proxyd/backend.go

```diff
@@ -316,7 +325,16 @@ func (b *Backend) doForward(ctx context.Context, rpcReq *RPCReq) (*RPCRes, error
 		httpReq.SetBasicAuth(b.authUsername, b.authPassword)
 	}
 
+	xForwardedFor := GetXForwardedFor(ctx)
+	if b.stripTrailingXFF {
+		ipList := strings.Split(xForwardedFor, ",")
+		if len(ipList) > 0 {
+			xForwardedFor = ipList[0]
+		}
+	}
```

Comment:
If we're not stripping the XFF, we should add our IP to the end of the XFF list. We should define the IP as an environment variable, so that we can use Kubernetes `valueFrom` directives to set it in the cluster.

Reply:
If the env var isn't set, don't append the IP but warn about it in the logs.